services/ui-src/src/types/index.ts

```diff
@@ -169,6 +169,11 @@ export declare type EntityType =
   | "qualityMeasures"
   | "sanctions";
 
+export enum ModalDrawerEntityTypes {
+  ACCESS_MEASURES = "accessMeasures",
+  QUALITY_MEASURES = "qualityMeasures",
+  SANCTIONS = "sanctions",
+}
```

Comment:
Suggestion: Can probably combine these with the `EntityType` type above or something. Like maybe we make a DrawerEntityType that's just `plans` and `bssEntities` and then EntityType is the union of Drawer and ModalDrawer.

Reply:
👍